src/Controls/samples/Controls.Sample/Pages/Controls/PickerPage.xaml.cs

```diff
@@ -112,6 +115,42 @@ void UpdatePickerBindingContext()
 				}
 			};
 		}
+
+		void OnUpdateHorizontalOptionsClicked(object sender, EventArgs e)
+		{
+			AlignmentPicker.HorizontalOptions = GetLayoutOptions(_horizontalOptionsIndex);
+
+			_horizontalOptionsIndex++;
+
+			if (_horizontalOptionsIndex == 4)
+				_horizontalOptionsIndex = 0;
+		}
+
+		void OnUpdateVerticalOptionsClicked(object sender, EventArgs e)
+		{
+			AlignmentPicker.VerticalOptions = GetLayoutOptions(_verticalOptionsIndex);
+
+			_verticalOptionsIndex++;
+
+			if (_verticalOptionsIndex == 4)
+				_verticalOptionsIndex = 0;
+		}
```

Comment:
We can improve things an little more with a method that doe the logic to update an index via a ref parameter:

```suggestion
       void UpdateIndex (ref int index) {
           index++;
           if (index == 4)
               index = 0;
       }
       
		void OnUpdateHorizontalOptionsClicked(object sender, EventArgs e)
		{
			AlignmentPicker.HorizontalOptions = GetLayoutOptions(_horizontalOptionsIndex);
            UpdateIndex(_horizontalOptionsIndex);
		}

		void OnUpdateVerticalOptionsClicked(object sender, EventArgs e)
		{
			AlignmentPicker.VerticalOptions = GetLayoutOptions(_verticalOptionsIndex);
			UpdateIndex(_verticalOptionsIndex);
		}
```

Reply:
Github, thanks for messing the indentation...